tests/phpunit/unit-tests/Controller/ActionTest.php

```diff
@@ -1,14 +1,11 @@
 <?php
 
-namespace GFPDF\Tests;
+namespace GFPDF\Tests\Controller;
```

Comment:
Ideally, we should use the same namespace as the class being tested.

Reply:
Fixed on recent push.